packages/sdk-react/src/components/payables/PayableDetails/PayableDetailsForm/PayableDetailsForm.tsx

```diff
@@ -234,6 +252,16 @@ const PayableDetailsFormBase = forwardRef<
 
     const className = 'Monite-PayableDetailsForm';
 
+    const isFieldRequired = (fieldName: keyof typeof defaultRequiredFields) => {
```

Comment:
A very similar function with the same name can be found in PayableDetailsInfo.tsx. Can we keep only one function?